tracing-subscriber/Cargo.toml

```
@@ -21,9 +21,9 @@ keywords = ["logging", "tracing", "metrics", "subscriber"]
 
 default = ["env-filter", "smallvec", "fmt", "ansi", "chrono", "tracing-log"]
 env-filter = ["matchers", "regex", "lazy_static"]
-fmt = ["owning_ref"]
+fmt = ["owning_ref", "registry"]
```

i think the `owning_ref` dep is no longer used --- can we remove that?
```suggestion
fmt = ["registry"]
```

---

(it would have to be removed on line 44 as well: https://github.com/tokio-rs/tracing/pull/420/files#diff-d417aef7dda8bc6c7eb19ebf8bd38a3cR44)